### PR TITLE
kernel-module-imx-gpu-viv: Fix build with gcc13

### PR DESCRIPTION
**Pull request by kraj**
Signed-off-by: Khem Raj <raj.khem@gmail.com>

### CODE REVIEW

File: recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv_6.4.3.p4.6+fslc.bb [hunk 1]
```diff
@@ -10,7 +10,8 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=12f884d2ae1ff87c09e5b7ccc2c4ca7e"
 PV .= "+git${SRCPV}"
 
 SRCREV = "1adf982c5954f2f87919c6874279386f82e898a6"
-SRC_URI = "git://github.com/Freescale/kernel-module-imx-gpu-viv.git;protocol=https;branch=master"
+SRC_URI = "git://github.com/Freescale/kernel-module-imx-gpu-viv.git;protocol=https;branch=master \
+           file://0001-Change-return-type-of-_QuerySignal-from-gctBOOL-gceS.patch"
```
Comment (otavio) [inline recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv_6.4.3.p4.6+fslc.bb:14]: Please send a PR to the repo so we avoid adding patch files.

**Review by otavio — CHANGES_REQUESTED** [review]

File: recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv_6.4.3.p4.6+fslc.bb (hunk 1 above)
Comment (kraj) [reply recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv_6.4.3.p4.6+fslc.bb:14]: done

File: recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv/0001-Change-return-type-of-_QuerySignal-from-gctBOOL-gceS.patch [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+/mnt/b/yoe/master/build/tmp/work/imx8qm_var_som-yoe-linux/kernel-module-imx-gpu-viv/6.4.3.p4.6+fslc+gitAUTOINC+1adf982c59-r0/git/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/gc_hal_kernel_os.c:5875:1: error: conflicting types for '_QuerySignal' due to enum/integer mismatch; have 'gceSTATUS(struct _gckOS *, void *)' {aka 'enum _gceSTATUS(struct _gckOS *, void *)'} [-Werror=enum-int-mismatch]
+ 5875 | _QuerySignal(
+      | ^~~~~~~~~~~~
+In file included from /mnt/b/yoe/master/build/tmp/work/imx8qm_var_som-yoe-linux/kernel-module-imx-gpu-viv/6.4.3.p4.6+fslc+gitAUTOINC+1adf982c59-r0/git/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/gc_hal_kernel_os.c:56:
+/mnt/b/yoe/master/build/tmp/work/imx8qm_var_som-yoe-linux/kernel-module-imx-gpu-viv/6.4.3.p4.6+fslc+gitAUTOINC+1adf982c59-r0/git/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/gc_hal_kernel_linux.h:322:1: note: previous declaration of '_QuerySignal' with type 'gctBOOL(struct _gckOS *, void *)' {aka 'int(struct _gckOS *, void *)'}
+  322 | _QuerySignal(
+      | ^~~~~~~~~~~~
+cc1: all warnings being treated as errors
+
+Upstream-Status: Submitted [https://github.com/Freescale/kernel-module-imx-gpu-viv/pull/34]
```
Comment (otavio) [inline recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv/0001-Change-return-type-of-_QuerySignal-from-gctBOOL-gceS.patch:19]: Merged. Please bump the `SRCREV.`

Comment (kraj) [reply recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv/0001-Change-return-type-of-_QuerySignal-from-gctBOOL-gceS.patch]: done

**Review by otavio — APPROVED** [review]

**Pull request merged**